### ISSUE TITLE
Broken dependency in version 1.1.1

### ISSUE THREAD

**Issue by jurgenhaas** [wong2/pick#61]
This version requires `dataclasses` which is not declared as a dependency.

This breaks all existing installations, if they update to this latest version.

**Comment by wong2** [comment]
Hi, what version of Python are you using?

**Comment by jurgenhaas** [comment]
I'm managing a lot of hosts, depending on their age they use between Python 3.6.9 and Python 3.8.10

**Comment by wong2** [comment]
The problem should affect 3.6.x, I'll fix it soon, sorry

**Comment by jurgenhaas** [comment]
Thanks, great. But I saw the same issue on 3.8.10 as well.

**Comment by jurgenhaas** [comment]
Yes, if `dataclasses` is installed. But that's not available by default and needs to be declared as a dependency. I wonder if it was already installed for your tests?

**Comment by wong2** [comment]
`dataclasses` should be a build-in lib since 3.7? https://docs.python.org/3/library/dataclasses.html

**Comment by jurgenhaas** [comment]
That's strange. I happen to have a fairly new Laptop here with Ubuntu 20.04 installed a couple of weeks ago. It has Python 3.8.10 installed but no `dataclasses` available. I have to do `pip3 install dataclasses` to make it work.

**Comment by wong2** [comment]
<img width="258" alt="截屏2021-12-06 下午11 18 57" src="https://user-images.githubusercontent.com/321947/144872111-a046465f-1749-4d88-b1ce-af3c1f882265.png">

The ci shows that only python 3.6 is not working

**Comment by jurgenhaas** [comment]
Well, that's fine. I can only report what I see on real hardware. I can't explain where the difference is coming from.

**Comment by wong2** [comment]
Version 1.2.0 has been released, which added `dataclasses` dependency for python 3.6.x

**Comment by wong2** [comment]
https://github.com/wong2/pick/runs/4432366828?check_suite_focus=true Test on ubuntu 20.04 also passed, so it's weird...
